.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,42 @@
+name: 🐛 Bug
```

nit: Previously it was "Bug report 🐞", which I think reads nicer (plus emoji on right side to keep it consistent with "Feature request 💄")

```suggestion
name: Bug report 🐞
```